src/trans-word-level/property.cpp

```diff
@@ -115,7 +115,45 @@ void property(
       instantiate(p, c, no_timeframes, ns);
 
     auto handle = solver.handle(tmp);
-    prop_handles.push_back(std::move(handle));
+    obligations[c].push_back(std::move(handle));
```

Comment:
I guess this isn't really a hot path, but this could safely be written as
````c++
obligations.insert(c, {solver.handle(tmp)});
````
as each index is unique. I'm actually no longer sure why this needs to be `exprt::operandst` as value type when it can only ever be a single `exprt`?

Reply:
Give me a sec, there will be more than one expression per index, and they'll come separately.